src/encord_active/lib/encord/actions.py

```diff
@@ -41,14 +49,50 @@ def __init__(self, project_dir: Path):
             Path(ssh_key_path).expanduser().read_text(encoding="utf-8"),
         )
 
-        self.original_project = self.user_client.get_project(original_project_hash)
+    def init_original_project(self):
         try:
-            if self.original_project.project_hash == original_project_hash:
-                pass
+            self.original_project = self.user_client.get_project(self.original_project_hash)
+            if self.original_project.project_hash == self.original_project_hash:
+                return False
         except AuthorisationError:
-            raise AuthorisationError(
-                f'The user associated to the ssh key `{ssh_key_path}` does not have access to the project with project hash `{original_project_hash}`. Run "encord-active config set ssh_key_path /path/to/your/key_file" to set it.'
-            )
+            return False
+        return True
+
+    def _upload_item(
+        self, dataset, label_row_hash, data_unit_hash, data_unit_hashes, new_du_to_original, uploaded_data
```

Comment:
Ideally, we'd add types here 

```suggestion
        self,
        dataset: Dataset,
        label_row_hash: str,
        data_unit_hash: str,
        data_unit_hashes: set[str],
        new_du_to_original: dict[str, dict], 
        uploaded_data: set[str],
```

Specifically for `new_du_to_original` we should probably add a `TypedDict`/`NamedTuple`/`@dataclass` since we know the fields upfront.